pint_pandas/testsuite/test_pandas_extensiontests.py

```diff
@@ -42,9 +42,12 @@ def dtype():
     return PintType("pint[meter]")
 
 
-@pytest.fixture
-def data():
-    return PintArray.from_1darray_quantity(np.arange(start=1.0, stop=101.0) * ureg.nm)
+_base_numeric_dtypes = [float, int]
+_all_numeric_dtypes = _base_numeric_dtypes + [np.complex128]
+
+@pytest.fixture(params=_all_numeric_dtypes)
+def data(request):
+    return PintArray.from_1darray_quantity(np.arange(start=1.0, stop=101.0, dtype=request.param) * ureg.nm)
```

Comment:
Need to add _all_numeric_dtypes to the other fixtures, data_missing, data_for_twos... data_for_grouping